includes/connect/class-wc-stripe-connect.php

```diff
@@ -146,6 +222,18 @@ public function maybe_handle_redirect() {
 		 * @return stdObject|WP_Error OAuth's response result or WP_Error.
 		 */
 		private function save_stripe_keys( $result, $type = 'connect', $mode = 'live' ) {
+			if ( WC_Stripe_Helper::is_enhanced_debug_mode_enabled() ) {
```

Comment:
Is it worth stashing this result in a local variable so we're only applying the filters once per method?
```suggestion
			$enhanced_debugging_enabled = WC_Stripe_Helper::is_enhanced_debug_mode_enabled();
			if ( $enhanced_debugging_enabled ) {
```

Reply:
The methods are not called frequently, but it makes sense 👍🏼 

Changed in [e810053](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4765/commits/e810053b0a00cfc0c684f650932c013ad4d470ac).